src/Components/Color/ColorForm.jsx

```diff
@@ -69,8 +69,11 @@ export default function ColorForm({ addColor }) {
         defaultValue={contrastText}
         onChange={(value) => setContrastText(value)}
       />
-
-      <button type="submit">ADD COLOR</button>
+      {isEditing ? (
+        <button type="submit">UPDATE COLOR</button>
+      ) : (
+        <button type="submit">ADD COLOR</button>
+      )}
```

Comment:
Looks good, maybe you could add classes to style it later. I don't know :D

Reply:
yes, good point. I could create a different className